### PR TITLE
Add Dai to sNST migration hook

### PR DESCRIPTION
**Pull request by oskarvu**
Closes FRO-832

### CODE REVIEW

**Review by yivlad — APPROVED** [review]

File: packages/app/src/domain/migration-actions/constants.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const MIGRATE_ACTIONS_ADDRESS = '0xB08543E5Ba2a382dd38E84Fe12656Cd35A42e12B'
```
Comment (krzkaczor) [inline packages/app/src/domain/migration-actions/constants.ts:1]: why not somewhere in config dir? also this should have a comment to move it to wagmi config once deployed.

File: packages/app/src/domain/migration-actions/useMigrateDAIToSNTS.test.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    })
+    expect((result.current as any).error).toBeUndefined()
+
+    result.current.write()
+
+    await waitFor(() => {
+      expect(result.current.status.kind).toBe('success')
+    })
+
+    await waitFor(() => {
```
Comment (krzkaczor) [inline packages/app/src/domain/migration-actions/useMigrateDAIToSNTS.test.ts:106]: it this waitFor needed here?

File: packages/app/src/domain/migration-actions/useMigrateDAIToSNTS.test.ts [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        allowanceQueryKey({
+          token: dai,
+          spender: MIGRATE_ACTIONS_ADDRESS,
+          account,
+          chainId: mainnet.id,
+        }),
+      )
+    })
+
+    await waitFor(() => {
```
Comment (krzkaczor) [inline packages/app/src/domain/migration-actions/useMigrateDAIToSNTS.test.ts:117]: same

**Review by krzkaczor — APPROVED** [review]

File: packages/app/src/domain/migration-actions/useMigrateDAIToSNTS.test.ts (hunk 2 above)
Comment (oskarvu) [reply packages/app/src/domain/migration-actions/useMigrateDAIToSNTS.test.ts:106]: No. Removed also in other hooks.

**Pull request merged**